intelmq_manager/static/js/static.js

```diff
@@ -526,8 +526,8 @@ $(document).ready(function() {
                 $('#modalLoginForm').modal('hide');
                 updateLoginStatus();
             } else {
-                // If authentication failed, this error message is displayed.
-                $('#loginErrorField').text('Invalid username and/or password');
+                // If authentication failed, the returned error message is displayed.
+                $('#loginErrorField').text(data);
```

Comment:
```suggestion
                $('#loginErrorField').text(data.error);
```
This, plus changing the check above determining the request was successful from whether the `login_token` and `username` attributes are present to checking for the `error` attribute.

Reply:
Hm, but the absence of the error attribute does not imply the existence of the login_token and username attributes, as something else could have gone wrong too. (or the version of the API is <= 3.0.0)

Reply:
I'd leave it up to the API to return the correct response when it claims it does.
> (or the version of the API is <= 3.0.0)

The components should all be kept in sync, should they not? I do not think there is currently any expectation of the API being fully compatible forwards or backwards.